docs/can-define-validate-validatejs.md

```diff
@@ -0,0 +1,64 @@
+@module {function} can-define-validate-validatejs
+@parent can-ecosystem
+@group can-define-validate-validatejs/DefineMap.prototype DefineMap.prototype
+@package ../package.json
+
+@description Adds helpful validation methods and observables to a [can-define/map/map]
+using [validate.js](https://validatejs.org/).
+
+@signature `defineValidator(Map)`
+
+  Checks for ValidateJS constraints and attaches useful methods.
```

Comment:
Probably should list the methods.  Say `adds the methods to the Map prototype`.  It's ok to be VERY technical here.